File: server/tests/fixtures/alpha_rule_triggers/javascript/bug_boolean_bitwise_misuse.js
// Should trigger: bug.boolean_bitwise_misuse
function checkConditions(a, b, c, d) {
    if ((a === 1) & (b > 0)) {  // using bitwise & instead of logical &&
        return true;
    }
    if ((c < 10) | (d === 5)) {  // using bitwise | instead of logical ||
        return true;
    }
    return false;
}

File: server/tests/fixtures/alpha_rule_triggers/python/bug_boolean_bitwise_misuse.py
# Should trigger: bug.boolean_bitwise_misuse
def check_conditions(a, b, c, d):
    if (a == 1) & (b > 0):  # using bitwise & instead of logical and
        return True
    if (c < 10) | (d == 5):  # using bitwise | instead of logical or
        return True
    return False
